config/environments/production.rb

```diff
@@ -64,13 +64,13 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
   #devise config
-  config.action_mailer.default_url_options = { :host => 'localhost' }
+  config.action_mailer.default_url_options = { :host => Rails.application.secrets.devise_mailer_url }
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.smtp_settings = { :address => "localhost", :port => 25 }
-  config.action_mailer.default :from => 'hydranorth@mailinator.com'
+  config.action_mailer.default :from => Rails.application.secrets.devise_email
 
   #contact for config
-  config.contact_email = 'hydranorth@mailinator.com'
+  config.contact_email = Rails.application.secrets.contact_email
```

Comment:
Have a feeling this contact_email is never being used.

Looks like it was introduced here and nothing references it:
https://github.com/ualbertalib/HydraNorth/commit/de119a12d76ce0d2052e4986cce07ae4928b4b4f#diff-a2f09e1db7a47eb67c5849478a97f3d7